WpNginxConfig.conf

```diff
@@ -0,0 +1,44 @@
+server {
+    listen 443 ssl;
+    listen [::]:443 ssl;
+    include snippets/self-signed.conf;
+    include snippets/ssl-params.conf;
+
+
+    server_name 192.168.50.2;
+    root /var/www/wordpress;
+
+    index index.php index.html index.htm;
+
+    location / {
+        #try_files $uri $uri/ =404;
```

Comment:
what this line does?

Reply:
line 11 = nginx prioritize the serving file that is above, in this case index.php.

line 13/14 = checks for the existence of file matching a URI request. If not found, returns 404 error.